source/manage-apps/configure/config.txt

```diff
@@ -38,6 +38,14 @@ using structured JSON configuration and JavaScript source code files.
 - ``.js`` files define serverless application logic used in functions, triggers,
   HTTPS endpoints, and custom resolvers.
 
+.. important:: Reserved File Names
+   
+   You can include arbitrarily named files in your realm app config, with a few exceptions.
+   The following file names should not be used: ``stitch.json``, ``config.json``, 
+   ``realm_config.json``, and ``app_config.json``. Files with these names
+   conflict with pre-made files of the same name in their app structure's file
+   system. Using these file names throws an error when you import the app using the CLI.
```

Comment:
doesn't always throw error, only if used in a manner differently from the configuration's format.